public/about.html

```diff
@@ -36,7 +36,7 @@
     </header>
     <div class="main-content"> 
         <div class="header-image">
-            <img src="images/header-images/about-header.png" alt="About header image">
```

Comment:
Pet peeve of mine is child folders of the images directory which are called something-images. Oh we know they'll be images!